### PR TITLE
Addressing issues #2 and #4 on UI changes

### PR DESCRIPTION
**Pull request by francistse23**
Fixes Kedyn/fusliez-notes#2
Fixes Kedyn/fusliez-notes#4

- Added more padding to the player icon and name
- Added background color to match active players' colors (special case for brown since the default brown color looks more like burgundy)
- Added font weight change by changing the Google Font link to import different font weights (might need to adjust the heading/title font weights since it defaults to 400 now)
- Added font color variance in dark/light mode with contrasting text shadow to highlight player name

### CODE REVIEW

**Comment by francistse23** [comment]
very first attempt at contributing to something open sourced. feel free to leave any feedback.

thanks!

File: src/components/MainContent.tsx [hunk 1]
```diff
@@ -140,12 +143,19 @@ export default function MainContent(props: IMainContentProps): JSX.Element {
                       <Form.Control
                         type="text"
                         placeholder="Player name"
-                        size="sm"
-                        className={`my-1 ${dark ? "dark-input" : ""}`}
-                        defaultValue={player.name}
+                        size="md"
```
Comment (Kedyn) [inline src/components/MainContent.tsx:146]: md not an available size, sm or lg reference (https://react-bootstrap.github.io/components/forms/#form-control-props)

File: src/components/MainContent.tsx [hunk 2]
```diff
@@ -140,12 +143,19 @@ export default function MainContent(props: IMainContentProps): JSX.Element {
...
+                        className={`my-1 px-3 ${dark ? "dark-input" : ""}`}
+                        defaultValue={name}
                         onChange={(
                           event: React.ChangeEvent<HTMLInputElement>
                         ) => handleChange(section, index, event)}
+                        style={{
+                          color: dark ? "white" : "black",
+                          fontWeight: "600",
```
Comment (Kedyn) [inline src/components/MainContent.tsx:154]: minor change from string to number should fix the error warning.

**Comment by Kedyn** [comment]
Awesome stuff just fix those two things and we can merge

**Comment by francistse23** [comment]
just fixed it. thanks for reviewing! quick question, did you see those errors from GitHub directly? Or did you have to pull?

**Comment by Kedyn** [comment]
I pulled and vscode showed those errors

**Pull request merged**